### DISCUSSION TITLE
Date-Fns Issues: Documentation not working with V2, no download with V2, 1.30 seems last safe version

### DISCUSSION THREAD

**Discussion by fendrikat** [js-fns/date-fns#1938] [General]
Hi,

..I do like Date-Fns in general, but since V2 it became basically unusable:

- The documentation is not accessible anymore, no matter in any V2 which topic is clicked, there is never any documentation showing up. Now considering my Chrome is on the latest release and this issue with Date-Fns is as old as V2, I assume that is happily accepted for some reasons

- Download seems not to be possible anymore - I do not use NPM or any such, as much runs offline, it complicates matters a lot and I saw a number of websites having issues with them. All libraries I use are available for download, except Date-Fns.

So for this, V1.30 is the last working, safe version, which I use in my little projects, maybe one day the documentation will be accessible again and it can be downloaded again.

Frank

**Comment by michaelyuen** [comment]
I don't fully understand what you said. I don't know that I'd say I have used v2 extensively, but for what I have done it has worked well.

All the same, if you're looking to download the library directly would unpkg be an option? https://unpkg.com/browse/date-fns@2.16.1/
